docs/quality-assurance/safety-assurance.md

```diff
@@ -0,0 +1,98 @@
+# What is safety ?
+
+So the first question is what is safety in relation to UKHO products:
+
+* An error in a product that could potentially impact the safety of the mariner
+* A delay to the mariner that could potentially impact the safety of the mariner
```

Comment:
These don't answer the question "what is safety", they are  problems that need to be addressed by safety.